### PR TITLE
Add mysql_login

### PR DESCRIPTION
**Pull request by 422926799**
Add mysql_login

### CODE REVIEW

File: secistsploit/modules/auxiliary/mysql_login.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+             "www.422926799.github.io"
+        ),
+
+    }
+
+    rhost=OptIP("",'Target IP')
+    rport=OptPort(3306,"Target port")
+    usernaeme=OptString("root","Target username")
+    password=OptString("root","Target Password")
+    userfile=OptString("","Target username dict")
```
Comment (blue-bird1) [inline secistsploit/modules/auxiliary/mysql_login.py]: 也许应该使用 [OptWordlist](https://github.com/secistgroup/SecistSploit/blob/3bae0246948b70fdd4f40d291235e28064527f5f/secistsploit/core/exploit/option.py#L122)

File: secistsploit/modules/auxiliary/mysql_login.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            dk=open(userfile,'r')
+            for r in dk.readlines():
+                qc2="".join(r.split('\n'))
+                try:
+                    connect = pymysql.connect(host=rhost, port=rport, user=qc2, passwd=password)
+                    print('[+] Mysql Connect {}:{} username:{} password:{} ok'.format(rhost, rport, qc2, password))
+                    break
+                except:
+                    print('[-] Mysql Connect {}:{} username:{} password:{} found'.format(rhost, rport, qc2,password))
+        if rhost!='' and rport!='' and username=='' and userfile!='' and password=='' and passwdfile!='':
```
Comment (blue-bird1) [inline secistsploit/modules/auxiliary/mysql_login.py]: emmm 为什么不在最前面把`rhost`and`rport`判断了

File: secistsploit/modules/auxiliary/mysql_login.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        username=(self.usernaeme)
+        password=(self.password)
+        userfile=(self.userfile)
+        passwdfile=(self.passwdfile)
+        if rhost!='' and rport!='' and username!='' and password !='':
+            try:
+                connect=pymysql.connect(host=rhost,port=rport,user=username,passwd=password)
+                print('[+] Mysql Connect {}:{} username:{} password:{} ok'.format(rhost,rport,username,password))
+            except:
+                print('[-] Mysql Connect {}:{} username:{} password:{} found'.format(rhost,rport,username,password))
```
Comment (blue-bird1) [inline secistsploit/modules/auxiliary/mysql_login.py]: 确定不是 not found?

File: secistsploit/modules/auxiliary/mysql_login.py (hunk 2 above)
Comment (422926799) [reply secistsploit/modules/auxiliary/mysql_login.py]: 哈哈哈= =

Comment (422926799) [reply secistsploit/modules/auxiliary/mysql_login.py]: 谢谢青鸟啦

**Pull request merged**